### ISSUE TITLE
rest query language support

### ISSUE THREAD

**Issue by linux-code** [microsoft/spring-data-cosmosdb#528]
Hi,

We are having use case for search API and this search API can take list of inputs with name , operator and value. Could you please let me know if `spring-data-cosmosdb` supports rest query language?

**Comment by kushagraThapar** [comment]
I don't think it does. I have seen other customers building sql query from REST query and then calling search using the built sql query.

**Comment by kushagraThapar** [comment]
@linux-code - Were you able to use the workaround ?
Or is this something you need support on spring-data-cosmosdb ?
